custom_components/ramses_cc/services.yaml

```diff
@@ -302,14 +298,19 @@ set_zone_mode:
           mode: slider
 
     duration:
+      name: Duration
```

Comment:
Shall it be `Duration (hours/minutes)` too?

Reply:
I didn't change this one before as there isn't the peculiarity of having two separate fields that do more or less the same thing for this service.

Reply:
From what you were saying is it also the case that the underlying will actually respect durations to the second as well? I haven't checked.